js/game.js

```diff
@@ -249,30 +272,63 @@ function bestDicePairPlusAce(dice) {
     return [];
 };
 
-// Results step
-function showResults() {
-    game.step = Steps.RESULTS;
-    game.renderFun();
+function determineWinner(subgame, dice1, dice2) {
+    let bestDice1 = bestDice(subgame, dice1);
+    let bestDiceSum1 = bestDice1.reduce((acc, val) => acc + dice1[val], 0);
+    let bestDice2 = bestDice(subgame, dice2);
+    let bestDiceSum2 = bestDice2.reduce((acc, val) => acc + dice2[val], 0);
+    switch (subgame) {
+        case Subgames.BIGGEST:
+        case Subgames.PAIR_PLUS_ACE:
+            // Bigger = better
+            if (bestDiceSum1 == 0 && bestDiceSum2 == 0) {
+                // No game
+                return 0
+            } else if (bestDiceSum1 >= bestDiceSum2) {
+                // Ties go to first player
+                return 1;
+            } else {
+                return -1;
+            }
+        case Subgames.SMALLEST:
+        case Subgames.ONE_AS_TWO:
+            // Bigger = better
+            if (bestDiceSum1 == 0 && bestDiceSum2 == 0) {
+                // No game
+                return 0
+            } else if (bestDiceSum1 > 0 && bestDiceSum1 <= bestDiceSum2) {
+                // Ties go to first player
+                return 1;
+            } else {
+                return -1;
+            }
+    }
 };
 
 // Exports
-module.exports = {
-    // Enums
-    Bets: Bets,
-    Steps: Steps,
-    Subgames: Subgames,
-    // Objects
-    game: game,
-    // Functions
-    beginGame: beginGame,
-    bestDice: bestDice,
-    betCheck: betCheck,
-    betFold: betFold,
-    betPass: betPass,
-    betRaise: betRaise,
-    endRollingStep: endRollingStep,
-    finishSubgame: finishSubgame,
-    rerollDice: rerollDice,
-    rollInitialDice: rollInitialDice,
-    updateSelectedDie: updateSelectedDie
-};
+if (typeof module !== 'undefined' && typeof module.exports !== 'undefined') {
```

Comment:
omg thank god i couldn't figure out how to make this not show up!!

Reply:
The bad thing is that this brings the coverage percentage a smidgen down because the tests can't check the "else" branch. Uh huh, life's like this!